### PR TITLE
fix(web): more UI adjustments

### PR DESCRIPTION
**Pull request by dgdavid**
As we know, there still a bunch of things to improve/polish in the new UI. This set of changes address some of them, see the list of commits.

<details>
<summary>Show/hide relevant screenshots</summary>

---

Screenshot from 2024-06-13 10-30-50

Screenshot from 2024-06-12 23-18-38
Screenshot from 2024-06-12 23-19-21
</details>

### CODE REVIEW

File: web/src/components/overview/OverviewPage.jsx [hunk 1]
```diff
@@ -52,21 +49,24 @@ const IssuesList = ({ issues }) => {
...
+        />
       );
       list.push(link);
     });
   });
 
   return (
     <EmptyState
-      title={_("Before installing the system, you need to pay attention to the following tasks:")}
+      title={_("Before installing the system, you need to pay attention to the following tasks")}
```
Comment (imobachgs) [inline web/src/components/overview/OverviewPage.jsx]: This text changed here. If you want to drop the colon (:), I would replace it with a stop (.).

Having said that, I do not know if the empty state plus the alert might be too much.

Captura desde 2024-06-13 06-27-37

**Review by imobachgs — COMMENTED** [review]
Just a comment regarding the issues. Otherwise, LGTM.

File: web/src/components/overview/OverviewPage.jsx (hunk 1 above)
Comment (dgdavid) [reply web/src/components/overview/OverviewPage.jsx]: Something went wrong from my side when merging the "adjust wording" commit here. Will check.

Comment (dgdavid) [reply web/src/components/overview/OverviewPage.jsx]: > Having said that, I do not know if the empty state plus the alert might be too much.

Yes, it's too much. I was playing with a "NotificationDrawer" and also with some customs overrides, but I didn't found a convincing proposal yesterday. Let's try it again :)

Comment (dgdavid) [reply web/src/components/overview/OverviewPage.jsx]: Fixed by picking up an idea I played yesterday and @joseivanlopez proposed this morning: using the PF/NotificationDrawer component. However, I've overwritten some styles to make it fit better in the overview page.

As disccussed offline, there is room for more improvements, but the current status should be enough for merging to master and continue from there.

| Problems | No problems |
|-|-|
|Screenshot from 2024-06-13 10-30-50 | Screenshot from 2024-06-13 10-31-02 |

**Review by imobachgs — APPROVED** [review]

**Comment by dgdavid** [comment]
@balsa-asanovic 

Just in case you missed this... it's now merged. As said there, code-wise still a lot of work to do. But it's ready for getting feedback from user POV.

**Pull request merged**